test/check-machines-disks

```diff
@@ -954,6 +980,42 @@ class TestMachinesDisks(VirtualMachinesCase):
             expected_target='sdc',
         ).execute()
 
+        # Disk with serial number
+        self.VMAddDiskDialog(
+            self,
+            pool_name='myPoolOne',
+            volume_name='disk_with_serial',
+            mode='create-new',
+            serial='disk_WITH-serial+.',
+            expected_target='vdc',
+        ).execute()
+
+        # Disk with serial number
+        self.VMAddDiskDialog(
+            self,
+            pool_name='myPoolOne',
+            volume_name='disk_with_serial_warning1',
+            mode='create-new',
+            serial='íňválíď;šéříáľ123',
+            expected_serial='vl123',
+            expected_target='vdd',
+            xwarning_object="serial-characters",
+            xwarning_message="Allowed characters: basic Latin alphabet, numbers, and limited punctuation (-, _, +, .)"
```

Comment:
Repeating the whole message here is a bit weird; as it uses substring matching anyway, "Allowed characters" seems enough here.